packages/frontend/src/utils/twoFactor/twoFactorBase.js

```diff
@@ -0,0 +1,214 @@
+import { BN } from 'bn.js';
```

Comment:
This file was moved and renamed, the history is still here despite the PR showing it as new.